app/circleSettings.js

```diff
@@ -0,0 +1,148 @@
+					setModalVisible(!modalVisible);
+					}}>
+					<StyledView>
+						<StyledView>
+							<StyledText>Hello World!</StyledText>
+							<StyledText onPress={() => setModalVisible(!modalVisible)}>Hide Modal</StyledText>
+						</StyledView>
+					</StyledView>
+				</Modal>
+
+				<StyledView className='top-[130px] w-[85%] gap-y-8 flex'>
+					<StyledView className="bg-grey h-[60px] py-[9px] px-[50px] rounded-xl items-center flex flex-row">
+						<Button
+							btnStyles='inline bg-grey border-2 border-purple mr-3'
+							height={'h-[45px]'}
+							width={'w-[45px]'}
+							iconSize={30}
+							icon='musical-notes'
+							iconColor='white'
+						/>
+						<StyledText className='inline font-bold text-3xl text-offwhite'>
+							Circle Name
+						</StyledText>
+					</StyledView>
+					<StyledView className="bg-grey h-[60px] py-[12px] rounded-xl pl-5 flex flex-row">
+						<StyledText className='font-bold text-3xl text-offwhite'>
+							Notifications
+						</StyledText>
+						<TouchableOpacity onPress={toggleSwitch}>
+							<StyledView
+								className='left-[50px] pt-9 w-[80px] h-[30px] rounded-full'
+								style={{
+									backgroundColor: isEnabled ? "#00A55E" : '#F9A826',
+								}}
+							>
+								<StyledAnimatedView
+									className='absolute top-1 w-[28px] h-[28px] rounded-full bg-white	'
+									style={{
+										left: togglePosition,
+									}}
+								/>
+							</StyledView>
+						</TouchableOpacity>
+					</StyledView>
+					<StyledView className="bg-grey h-[425px] py-[12px] rounded-xl pl-5">
+						<StyledText className='font-bold text-3xl text-offwhite'>
+							Members
+						</StyledText>
+						<StyledScrollView className="">
+							<Member img={Trevor} name="Josh Philips" username="JoshuaP.149134" role="own"></Member>
+							<Member img={Trevor} name="Alex Muresan" username="muresanCoder.20" role="mod"></Member>
+							<Member img={Trevor} name="Nason Allen" username="AllenNasin0987654" role="mod"></Member>
+							<Member img={Trevor} name="Aidan Hubley" username="HubleyPraying" role="ban"></Member>
+							<Member img={Trevor} name="Trevor Bunch" username="BunchTrevoraccount" role="mem"></Member>
+							<Member img={Trevor} name="Another Account" username="ExampleAccount1" role="sus"></Member>
+							<Member img={Trevor} name="Another Account" username="ExampleAccount2" role="mem"></Member>
+							<Member img={Trevor} name="Another Account" username="ExampleAccount3" role="mem"></Member>
+							<Member img={Trevor} name="Another Account" username="ExampleAccount4" role="mem"></Member>
+							<Member img={Trevor} name="Another Account" username="ExampleAccount5" role="mem"></Member>
+							<Member img={Trevor} name="Another Account" username="ExampleAccount6" role="mem"></Member>
+							<Member img={Trevor} name="Last Account" username="ExampleAccount7" role="ban"></Member>
+						</StyledScrollView>
+					</StyledView>
+				</StyledView>
+
+				<Button
```

Comment:
I think that these buttons should be moved to outside of this view. They render really high on the page and rendering them directly inside of the SafeAreaView follow the way the other buttons are rendered

Reply:
Fixed